WordPress/src/main/java/org/wordpress/android/ui/posts/editor/StoriesEventListener.kt

```diff
@@ -95,6 +112,39 @@ class StoriesEventListener @Inject constructor(
         storySaveMediaListener = newListener
     }
 
+    fun postStoryMediaUploadedEvent(mediaModel: MediaModel) {
+        // if this is a Story media item, then make sure to keep up with the StoriesPrefs serialized slides
+        // this looks for the slide saved with the local id key (media.getId()), and re-converts it to
+        // mediaId.
+        // Also: we don't need to worry about checking if this mediaModel corresponds to a media upload
+        // within a story block in this post: we will only replace items for which a local-keyed frame has
+        // been created before, which can only happen when using the Story Creator.
+        // if this is a Story media item, then make sure to keep up with the StoriesPrefs serialized slides
+        // this looks for the slide saved with the local id key (media.getId()), and re-converts it to
+        // mediaId.
+        // Also: we don't need to worry about checking if this mediaModel corresponds to a media upload
+        // within a story block in this post: we will only replace items for which a local-keyed frame has
+        // been created before, which can only happen when using the Story Creator.
```

Comment:
@mzorz the comment here seems duplicated 😉 Thanks for expounding here though. The comments are appreciated.

Reply:
oops! addressed in d67f2fffe9